language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala

```diff
@@ -248,6 +269,23 @@ object ScalaSourceEntityExtractor {
   def define(n: Name): Env[Unit] =
     updateState(_.addDef(n))
 
+  def link(n: Name, targets: List[List[Name]]): Env[Unit] = {
+    if (targets.nonEmpty) {
```

Comment:
I don't think you need the `targets.nonEmpty` since traversing an empty list gives you just the empty `Env[Unit]`